artifacts/grafana-operator-manifests/kustomization.yaml

```diff
@@ -3,4 +3,9 @@ kind: Kustomization
 resources:
   - amg_grafana-identity.yaml
   - amg_grafana-amp-datasource.yaml
-  - amg_grafana-dashboard.yaml
+  - amg_grafana-cw-datasource.yaml
+  - amg_grafana-xray-datasource.yaml
+  - amg_grafana-dashboards-amp.yaml
```

Comment:
If you do this. All yamls will be deployed at one short even if the customer is not looking to java or ngnix monitoring. I have design in mind for this. You should have separate kustomization for this and have separate folder which can be sync'd by kustomization for individual monitoring needs. We should not club all under one `kustomization`.

Reply:
Awesome! TBH I didn't know how to separate the resources creation, I'll give a go at what you say if I understood well. But my goal is exactly what you say, deploy what customer wants and not everything